deployments/helm/sumologic-the-coffee-bar/values.yaml

```diff
@@ -3,20 +3,22 @@
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
-replicaCountClicker: 1
-replicaCountBar: 1
-replicaCountPostgres: 2
-
+replicaCountClicker: 3
```

Comment:
There's no need to increase number of replicas for demo apps to present basic capabilities